src/pocketmine/event/block/SignChangeEvent.php

```diff
@@ -79,7 +79,7 @@ public function getLine($index){
 	 * @param string[] $lines
 	 */
 	public function setLines(array $lines){
-		if(count($lines) > 4){
+		if(count($lines) === 4){
```

Comment:
It's obvious, in any way the setLines() will overwrite all the lines. If I need to edit just "x" lines I will use setLine(), but if need to create a sign with 2 lines, with my way the developer just need to do "setLines(array("line1", "line1"));", with the actually way he does "setLines(array("line1", "line2", "", "");"

True, array_pad() is better xD